adlfs/multithread.py

```diff
@@ -14,24 +14,20 @@
 
 Only implements upload and download of (massive) files and directory trees.
 """
-from concurrent.futures import ThreadPoolExecutor, wait
 import glob
 import logging
-import multiprocessing
 import os
-import pickle
-import time
-import uuid
 
 from .core import AzureDLPath
+from .transfer import ADLTransferClient
 from .utils import commonprefix, datadir, read_block, tokenize
 
 MAXRETRIES = 5
 
 logger = logging.getLogger(__name__)
 
 
-class ADLDownloader:
+class ADLDownloader(object):
```

Comment:
No, this denotes the parent class. It makes no difference in py3, but has subtle implications in the setup and teardown mechanism for instances for older pythons. It's a "can't hurt, might help" situation and doesn't chance the calling signature.